test/streamlit.py

```diff
@@ -0,0 +1,64 @@
+import os
+import argparse
+import time
+import librosa
+import soundfile
+import streamlit as st
+import torch
+from io import BytesIO
+from voicefixer import VoiceFixer
+
+
+@st.experimental_singleton
+def init_voicefixer():
+    return VoiceFixer()
+
+
+# init with global shared singleton instance
+voice_fixer = init_voicefixer()
+
+
+sample_rate = 44100  
+
+
+st.write('Wav player')
+
+
+w = st.file_uploader('Upload a wav file', type='wav')
+
+
+if w:
+    st.write('Inference : ')
+    
+    # choose options
+    mode = st.radio('Voice fixer mode (0: rm high frequency, 1: none, 2: train fixer)', [0, 1, 2])
+    if torch.cuda.is_available():
+        is_cuda = st.radio('Turn on GPU', [True, False])
+        if is_cuda != list(voice_fixer._model.parameters())[0].is_cuda:
+            device = 'cuda' if is_cuda else 'cpu'
+            voice_fixer._model = voice_fixer._model.to(device)
```

Comment:
Maybe here we need another else branch? I test it on my MacBook without GPU and it shows is_cuda is not defined in line 47.

```python3
    if torch.cuda.is_available():
        is_cuda = st.radio('Turn on GPU', [True, False])
        if is_cuda != list(voice_fixer._model.parameters())[0].is_cuda:
            device = 'cuda' if is_cuda else 'cpu'
            voice_fixer._model = voice_fixer._model.to(device)
    else:
        is_cuda = False
```

Reply:
OK, I'll merge it first and solve it by myself. Thanks for your work!

Reply:
Ah.. Because my laptop has a GPU, I cannot found that point. Thanks!